### ISSUE TITLE
error: Failed to get compiled source code of https://deno.land/std@0.59.0/fs/mod.ts. Reason: The system cannot find the path specified. (os error 3)

### ISSUE THREAD

**Issue by avudaiappan-cse** [deligenius/view-engine#11]
Directory
/
&nbsp;&nbsp;app.ts
&nbsp;&nbsp;index.ejs

**app.ts**
<pre>
<code>
import { Application } from "https://deno.land/x/oak/mod.ts";
import {
  viewEngine,
  engineFactory,
  adapterFactory,
} from "https://deno.land/x/view_engine/mod.ts";

const ejsEngine = engineFactory.getEjsEngine();
const oakAdapter = adapterFactory.getOakAdapter();

const app = new Application();

app.use(viewEngine(oakAdapter, ejsEngine));

app.use(async (ctx, next) => {
  ctx.render("index.ejs", { data: { name: "John" } });
});
</code>
</pre>

await app.listen({ port: 8000 });

**index.ejs**
<code>
&lt;body&gt;
  Hobbies of <%=data.name%>
&lt;/body&gt;
</code>

**Deno version**
deno 1.1.2

**Comment by gjuoun** [comment]
I suspect that deno official website it's broken again.

Need to find an alternative package source.

**Comment by gjuoun** [comment]
As release v1.2.0 mentioned, the `view-engine v1.2.0 is bounded to oak v5.3.1`

Please use it as follow 
```ts
// app.ts
import { Application } from "https://deno.land/x/oak@v5.3.1/mod.ts";
import {
  viewEngine,
  engineFactory,
  adapterFactory,
} from "https://deno.land/x/view_engine@v1.2.0/mod.ts";

//... your code
```

**Comment by gjuoun** [comment]
Fixed @ 0ff656a

**Comment by gjuoun** [comment]
Tests passed in macOS and Windows 10 but failed in `ubuntu`. 
I assume it's Deno's compiler problem. Further investigation is needed.
